valhalla/meili/viterbi_search.h

```diff
@@ -10,6 +10,10 @@
 #include <valhalla/meili/priority_queue.h>
 #include <valhalla/meili/stateid.h>
 
+// undef win32 macro (if it exists)
+#ifdef SearchPath
```

Comment:
I wouldnt be opposed to changing the name if we have a name collision here.

Reply:
went ahead and renamed to SearchPathVS - fixes the naming conflict.